app/scripts/pages/bitswap.js

```diff
@@ -1,44 +1,51 @@
 import React from 'react'
 import FileList from '../views/filelist'
 import i18n from '../utils/i18n.js'
-import {Row, Col, Panel} from 'react-bootstrap'
-
-export default React.createClass({
-  displayName: 'Bitswap',
-  propTypes: {
-    pollInterval: React.PropTypes.func
-  },
-  getInitialState: function () {
-    var t = this
-
-    var update = function () {
-      t.props.ipfs.send('bitswap/wantlist', null, null, null, function (err, res) {
-        if (err) return console.error(err)
-        t.setState({ wantlist: res })
-      })
-    }
-
-    update()
-    t.props.pollInterval = setInterval(update, 1000)
-
-    return {
+import {
+  Row, Col, Panel
+}
+from 'react-bootstrap'
+
+export
+default class extends React.Component {
```

Comment:
Lets give the classes names as well `default class Bitswap extends React.Component`